sys-power/powerstat/files/powerstat-0.02.17-makefile.patch

```diff
@@ -0,0 +1,27 @@
+--- a/Makefile	2018-06-10 23:10:52.272789684 +0300
++++ b/Makefile	2018-06-10 23:11:51.673821892 +0300
+@@ -36,9 +36,6 @@
+ powerstat: powerstat.o
+ 	$(CC) $(CFLAGS) $< -lm -o $@ $(LDFLAGS)
+ 
+-powerstat.8.gz: powerstat.8
```

Comment:
You don't really have to remove this rule; it won't do anything if you remove the dep from `install` target. Avoid doing unnecessary stuff because it makes patches larger and less likely to apply.

Reply:
Ok.